### PR TITLE
[plugin.video.skyvideoitalia] 1.0.0

### PR DESCRIPTION
**Pull request by nixxo**
### Description
This addon shows video content from the website of Sky Italia (video.sky.it). Content is freely available, no login required.
News, sport, entertainment and such.

### Checklist:
- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.skyvideoitalia/resources/lib/skyitalia.py [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+    TOKEN = 'F96WlOd8yoFmLQgiqv6fNQRvHZcsWk5jDaYnDvhbiJk'
+    TIMEOUT = 15
+    LOGLEVEL = int(addonutils.getSetting('LogLevel'))
+    QUALITY = addonutils.getSetting('Quality')
+    QUALITIES = ['web_low_url', 'web_med_url', 'web_high_url', 'web_hd_url']
+    LOGOSDIR = '%sresources\\logos\\' % addonutils.PATH_T
+    FANART = addonutils.FANART
+
+    def __init__(self):
+        socket.setdefaulttimeout(self.TIMEOUT)
```
Comment (basrieter) [inline plugin.video.skyvideoitalia/resources/lib/skyitalia.py]: Why do you set the socket here? It's not used anywhere? And you already set the timeout at the urlopen level?

**Review by basrieter — COMMENTED** [review]
Please see my comments

File: plugin.video.skyvideoitalia/resources/lib/skyitalia.py (hunk 1 above)
Comment (nixxo) [reply plugin.video.skyvideoitalia/resources/lib/skyitalia.py]: you're right. It's leftover code before I put the timeout in the urlopen... I'll fix it.

Comment (basrieter) [reply plugin.video.skyvideoitalia/resources/lib/skyitalia.py]: Sure, give me a ping if it's done.

**Comment by nixxo** [comment]
@basrieter fixed.

**Review by basrieter — APPROVED** [review]

**Pull request merged**